internal/app/dogma/go.sum

```diff
@@ -23,6 +23,8 @@ go.linecorp.com/centraldogma v0.0.0-20181129085633-1c98e8bf157d h1:6OJCQUcIlPTei
 go.linecorp.com/centraldogma v0.0.0-20181129085633-1c98e8bf157d/go.mod h1:2oG/mC9W3NbCF7JlWjenR95+L7ZLBDMDrWsJGtLfTb0=
 go.linecorp.com/centraldogma v0.0.0-20181205125806-78f3817ddcb9 h1:JqEINY0HPCT8W7WzbZjcZvew+SGYrf8vcy1741EY2fY=
```

Comment:
Just curious. Don't we need to remove checksums for old versions?

Reply:
Remove them. Didn't know that there's a command for this which is `go mod tidy`. :)